src/pages/events/index.js

```diff
@@ -1,36 +1,87 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { graphql } from 'gatsby';
-import { Box, Heading } from 'grommet';
-
-import { BlogCard, Layout, SEO } from '../../components';
+import { Box, Heading, Button } from 'grommet';
+import {
+  Layout,
+  SEO,
+  PageDescription,
+  Card,
+  EventCard,
+  PastEventCard,
+} from '../../components';
 import { useSiteMetadata } from '../../hooks/use-site-metadata';
-
 function Events({ data }) {
-  const posts = data.allMarkdownRemark.edges;
+  const pastEvents = data.pastEvents.edges;
+  const upcomingEvents = data.upcomingEvents.edges;
   const siteMetadata = useSiteMetadata();
   const siteTitle = siteMetadata.title;
-  const currentDate = new Date().toISOString();
+
+  // Create box for each past event
+  const listPastEventBoxes = pastEvents.map(({ node }) => (
+    <EventCard
+      key={node.id}
+      category={node.frontmatter.category}
+      title={node.frontmatter.title}
+      timeframe={node.frontmatter.timeframe}
+      width={node.frontmatter.width}
+      content={node.rawMarkdownBody}
+      link={node.frontmatter.link}
+      image={node.frontmatter.image}
+    />
+  ));
 
   return (
     <Layout title={siteTitle}>
       <SEO title="Events" />
-      <Box flex overflow="auto" gap="medium" pad="small">
-        <Box flex={false} direction="row-responsive" wrap>
-          <Box pad={{ vertical: 'large', horizontal: 'large' }}>
-            <Heading margin="none">Events</Heading>
-          </Box>
-          <Box>
-            {posts.map(({ node }) => {
-              if (
-                node.fields.slug !== '/' &&
-                node.frontmatter.dateEnd > currentDate
-              ) {
-                return <BlogCard key={node.id} node={node} />;
-              }
-              return false;
-            })}
-          </Box>
+      <Box flex overflow="auto" pad="xlarge" wrap>
+        <PageDescription image="/img/events/EventsPage.svg" title="Events">
+          <Button
+            color="blue"
+            primary
+            label="Keep Me Posted"
+            href="/newsletter-signup"
```

Comment:
this is an internal route and doing it as an href here may be causing the entire page to get completely reloaded from the server rather than just doing the route. That's why we tend to use something that does history.push() or Gatsby's Link instead